### ISSUE TITLE
Finalize Time and Location of Data Viz Hacky Hour

### ISSUE THREAD

**Issue by rchampieux** [BioData-Club/BioData-Club.github.io#26]
This is scheduled for Friday July 27 and we usually start hacky hours at 4:00 or 4:30.  Alex and Eric should pick a time/location that suits them.

**Comment by erictleung** [comment]
Were we thinking of having this on Fri July 27th? Or is this still up in the air?

**Comment by rchampieux** [comment]
Hi @erictleung , that's correct.  But it's the location I wanted to make sure on.  Should we just do elephants again?

**Comment by ericearl** [comment]
Yes, consistency is good.  So 4pm July 27th at Elephant's Deli on the South Waterfront, right?  https://www.elephantsdeli.com/locations/south-waterfront/

**Comment by laderast** [comment]
sounds good to me
